thoth/storages/graph/postgres.py

```diff
@@ -267,10 +265,14 @@ def connect(self):
             raise AlreadyConnected("Cannot connect, the adapter is already connected")
 
         echo = bool(int(os.getenv("THOTH_STORAGES_DEBUG_QUERIES", 0)))
+        #TODO: Remove once https://github.com/kvesteri/sqlalchemy-utils/pull/372 is merged and new release of sqlalchemy_utils is released
+        is_successfully_started = False
         try:
             self._engine = create_engine(self.construct_connection_string(), echo=echo)
             self._sessionmaker = sessionmaker(bind=self._engine)
-        except Exception:
+            self._is_successfully_started = True
```

Comment:
Is there a need for a new object attribute? If so it should go to the `__init__` if we want to use it in other methods. Otherwise it can be a variable.

Reply:
Removed and implemented methods in `postgres_utils.py` :)